youtube_comedy_slam/youtube_comments_api.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python3
+
+import httplib2
+import os
+import sys
+import pprint
+import csv
+
+from apiclient.discovery import build
+from apiclient.errors import HttpError
+from oauth2client.client import flow_from_clientsecrets
+from oauth2client.file import Storage
+from oauth2client.tools import argparser, run_flow
+
+# CLIENT_SECRETS_FILE = "client_secrets.json"
+DEVELOPER_KEY = "AIzaSyAzV-ojzFgf3NSi9V8v9f_Ew2xQcRHrF84"
+YOUTUBE_API_SERVICE_NAME = "youtube"
+YOUTUBE_API_VERSION = "v3"
+
+youtube = build(YOUTUBE_API_SERVICE_NAME, YOUTUBE_API_VERSION, developerKey=DEVELOPER_KEY)
+
+
+
+
+def get_comment_threads(youtube, video_id, csvfile):
+    """
+    Our purpose is to download comments and append them to a csv with the following format:
+    videoId,textDisplay, canReply, totalReplyCount,etag,id,authorChannelId,authorDisplayName,likeCount,publishedAt,updatedAt,viewerRating
+
+    """
+    results = youtube.commentThreads().list(
+        part="snippet,replies",
+        textFormat="plainText",
+        maxResults=100,
+        videoId=video_id
+    ).execute()
+
+
+    # pp = pprint.PrettyPrinter(indent=4)
```

Comment:
The best practice is to let commented code without explanation.
So two possibilities:

1. Explain in some comment why you need to have here commented code
2. Delete the commented code